tests/components/somfy_mylink/test_somfy_shade.py

```diff
@@ -0,0 +1,16 @@
+import pytest
+
+from unittest.mock import Mock
+
+from homeassistant.components.somfy_mylink.cover import SomfyShade
+from somfy_mylink_synergy import SomfyMyLinkSynergy
+
+
+def test_default_state():
+    mock_mylink = Mock(spec=SomfyMyLinkSynergy)
+    somfy_shade = SomfyShade(mock_mylink, "target-id")
+
+    assert somfy_shade.state is None
+    assert somfy_shade.is_closed is None
+    assert somfy_shade.is_closing is None
+    assert somfy_shade.is_opening is None
```

Comment:
While it is good to add tests, we want tests for integrations to test with states on state machine, not test internal objects. So the integration should be set up and the resulting hass.states values should be checked.

Reply:
I don't think the existing tests have any coverage of the states of the shades -- should I remove this test, or would you prefer I figure out how to get test coverage of this?

Reply:
Remove it from this pull for now. We would very much appreciate tests so if you can work on that separate it would be much appreciated.

Reply:
Thanks! I made sure to manually test this on a local hass server on my machine, and it seems to work